spec/support/corpus.rb

```diff
@@ -134,15 +140,31 @@ def install_mutant
         system(%w[bundle])
       end
 
-      # Not in the docs. Number from chatting with their support.
-      CIRCLE_CI_CONTAINER_PROCESSES = 2
+      # The effective ruby file paths
+      #
+      # @return [Array<Pathname>]
+      def effective_ruby_paths
+        paths = Pathname
+          .glob(repo_path.join(RUBY_GLOB_PATTERN))
+          .sort_by(&:size)
+          .reverse
+
+        paths - excluded_paths
+      end
+
+      # The excluded file paths
+      #
+      # @return [Array<Pathname>]
+      def excluded_paths
+        Pathname.glob(repo_path.join(EXCLUDE_GLOB_FORMAT % exclude.join(',')))
+      end
 
       # Number of parallel processes to use
       #
       # @return [Fixnum]
       def parallel_processes
```

Comment:
Its a helper method, and I do not want to make it part of the public API of the corpus class just to avoid having it as a helper method.

No Public API > "Bad" private API.